src/mscorlib/src/System/Reflection/MemberSerializationStringGenerator.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Reflection;
+using System.Diagnostics;
+using System.Collections.Generic;
+
+namespace System
+{
+    internal static class MemberSerializationStringGenerator
+    {
+        //
+        // Generate the "Signature2" binary serialization string for PropertyInfos
+        //
+        // Because the string is effectively a file format for serialized Reflection objects, it must be exactly correct. If missing
+        // metadata prevents generating the string, this method throws a MissingMetadata exception.
+        // 
+        public static string SerializationToString(this PropertyInfo property) => ((RuntimePropertyInfo)property).SerializationToString();
```

Comment:
While I don't want to do it as part of _this_ PR, it is possible to share MemberSerializationStringGenerator.cs as well (CoreRT implements SerializationToString() as portable extension methods.)

The dead-code cleanup job on the CoreCLR side will be a bit involved as its SerializationToString() weaves in and out of FCalls and the normal ToString() path in a rather awkward dance.